lib/TransactionRunner.js

```diff
@@ -38,7 +38,7 @@ class TransactionRunner {
 
   config(config) {
     this.configuration = config;
-    this.multiBlueprint = Object.keys(this.configuration.data).length > 1;
+    this.multiBlueprint = this.configuration.apiDescriptions.length > 1;
```

Comment:
I didn't bother refactoring this part now. The `multiBlueprint` is something used only for the following code further down the file, and I didn't have the time to figure out all the consequences of it yet:

```js
    // Backward compatible transaction name hack. Transaction names will be
    // replaced by Canonical Transaction Paths: https://github.com/apiaryio/dredd/issues/227
    if (!this.multiBlueprint) {
      transaction.name = transaction.name.replace(`${transaction.origin.apiName} > `, '');
    }
```